db/schema.rb

```diff
@@ -43,8 +43,6 @@
     t.boolean  "fogged"
   end
 
-  add_index "checkins", ["device_id"], name: "index_checkins_on_device_id", using: :btree
```

Comment:
Not sure how these got deleted but you probably need to run rake:db migrate unless there is some reasoning behind it.